config/default.yml

```diff
@@ -2,9 +2,8 @@
 FactoryBot:
   Enabled: true
   Include:
-    - "**/spec/factories.rb"
+    - "**/factories.rb"
```

Comment:
But would we also analyze eg app/presenters/factories.rb?

Reply:
Yes, but I think this is a risk we should accept when we decide to put `**/` at the beginning of `Include`. It is also inconsistent that other `Include`s support it, but not only here.
Since it is rare for a constant to be defined in the plural form, I believe that this is rarely a problem in practice.

Reply:
I understand that factory_bot defaults to **/factories.rb as the definition file path. However, I doubt that it should be the default for rubocop-factory_bot. Because I am concerned about the increased risk of false positives, as also https://github.com/rubocop/rubocop-factory_bot/pull/104#discussion_r1494435518.

Also, even if some users wanted to fix this, I felt it was not necessary to include it in the default because it could be avoided by simply adding the target path to `FactoryBot: Include: `.